base/linalg/lapack.jl

```diff
@@ -5,11 +5,15 @@ module LAPACK
 
 const liblapack = Base.liblapack_name
 
-import Base.blasfunc
+import Base.@blasfunc
 
 import ..LinAlg: BlasFloat, Char, BlasInt, LAPACKException,
     DimensionMismatch, SingularException, PosDefException, chkstride1, chksquare
 
+function __init__()
+    const global VERSION = VersionNumber(laver()...)
```

Comment:
Can you explain a bit more? When is the constant set? It should correspond to the version actually linked when Julia is launched. What is the right solution?